src/index.js

```diff
@@ -1,27 +1,30 @@
 import _ from 'lodash';
-import {checkFlowFileAnnotation} from './utilities';
+import {
+  checkFlowFileAnnotation
+} from './utilities';
+import booleanStyle from './rules/booleanStyle';
```

Comment:
This is just sorting of the imports in an alphabetical order, as the order has not been kept.